packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -325,3 +325,39 @@ export function getIsOrgVerified(
     isInOrgScope: false,
   } as { isInOrgScope: false; orgVerified: never; autoAcceptEmailDomain: never };
 }
+
+export async function createAndAutoJoinIfInOrg({
+  team,
+  role,
+  invitee,
+}: {
+  team: TeamWithParent;
+  invitee: User;
+  role: MembershipRole;
+}) {
+  if (invitee.organizationId && invitee.organizationId !== team.parentId) {
+    return {
+      autoJoined: false,
+    };
+  }
+
+  if (!team.parentId) {
```

Comment:
What if he joins the main organisation team?

Reply:
then parentId is null and this is skipped - we want them to have the full onboarding flow. 

if autojoined here is true we skip the invite etc